Frameworks/QuartzCore/CATransaction.mm

```diff
@@ -36,14 +36,15 @@
 
```

Comment:
I don't want to change this now, but I'll add a TODO in the code.

---
In reply to: [86900474](https://github.com/Microsoft/WinObjC/pull/1318#discussion_r86900474) [](ancestors = 86900474)